packages/embark-contracts-manager/src/index.js

```diff
@@ -123,7 +123,8 @@ class ContractsManager {
 
           self.events.request("blockchain:contract:create", {abi: contract.abiDefinition, address: contract.deployedAddress}, async (contractObj) => {
             try {
-              const value = req.body.value !== undefined ? web3.utils.toWei(req.body.value, 'ether') : 0;
+              let value = typeof req.body.value === "number" ? req.body.value.toString() : req.body.value;
```

Comment:
What happens now if you send an undefined value. Seems like you pass `undefined` to `toWei`, does it behave correctly?

Reply:
As mentioned in the commit message

> The value is always sent as number and in fact always defined,

part of the store in the client is completely sent to the API so `req.body.value` is always defined